### PR TITLE
SCSS for existing components

### PR DESCRIPTION
**Pull request by wintondeshong**
Ported ITSCSS SCSS structure for existing atom components

- [ ] **n/a** Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] **n/a** No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [ ] **n/a** Localization: No hard-coded error messages in code files (_minimally_ in string constants)
- [x] New component and/ or properties? Make sure to update storybook

### CODE REVIEW

File: src/assets/scss/andculturecode-javascript-react-components.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * Class-based selectors which define undecorated design patterns, for example media object known from OOCSS.
+ *
+ * COMPONENTS
+ * All items that describe the physical attributes like buttons, sliders, cards, badges, etc...
+ *
+ * UTILITIES
+ * Utilities and helper classes with ability to override anything which goes before in the triangle, eg. hide helper class
+ */
+
+@import "0-vendor/__vendor", "1-settings/__settings", "2-tools/__tools",
```
Comment (brandongregoryscott) [inline src/assets/scss/andculturecode-javascript-react-components.scss:26]: Nitpick - I prefer breaking out an `@import "partial";` per line so it is easier to alphabetize

File: src/assets/scss/6-components/atoms/_buttons.scss [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+        }
+    }
+
+    &.-large {
+        @include padding(30px, 24px);
+        width: 100%;
+    }
+
+    &.-medium {
+        @include padding(11px, 24px);
```
Comment (jebediahelliott) [inline src/assets/scss/6-components/atoms/_buttons.scss:37]: Is this intentionally redundant with the default?

File: src/assets/scss/andculturecode-javascript-react-components.scss (hunk 1 above)
Comment (wintondeshong) [reply src/assets/scss/andculturecode-javascript-react-components.scss:26]: ah, i like that alternative being when comma-delimited it gets auto-formatted 👍

File: src/assets/scss/6-components/atoms/_buttons.scss (hunk 2 above)
Comment (wintondeshong) [reply src/assets/scss/6-components/atoms/_buttons.scss:37]: Good question. In porting some of these styles I have not had a chance to fully re-evaluate them. I'll at least leave a comment for later review.

**Comment by brandongregoryscott** [comment]
@all-contributors add @jebediahelliott for review

**Pull request merged**